lib/blacklight/solr/response/group_response.rb

```diff
@@ -39,6 +39,17 @@ def empty?
     total.zero?
   end
 
+  ##
+  # Overridden from Blacklight::Solr::Response::PaginationMethods to support
+  # grouped key specific i18n keys. `key` is the field being grouped
+  def entry_name(options)
+    begin
+      I18n.t("blacklight.entry_name.grouped.#{key}", raise: true)
```

Comment:
I think this can be collapsed using I18n fallbacks (and its pluralization methods?), something like:

```ruby
I18n.t("blacklight.entry_name.grouped.#{key}", default: 'blacklight.entry_name.grouped.default', count: options[:count])
```

Reply:
👍 I opted to leave the count out as that would require additional keys. Just using the rails helper for pluralization.